ingress/elb/elb.go

```diff
@@ -181,6 +190,13 @@ func (e *elb) Detach() error {
 	return nil
 }
 
+func (e *elb) Health() error {
+	if e.registeredFrontends != e.expectedFrontends {
```

Comment:
Thought about it a bit more - I think it does, since the goroutine that starts the http server for health checks may happen before the attach call. We can control that order in the main but I think it's safer just to assume health checks are thread safe.